src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
I think if the property exists at all it should be valid, so that rules the middle two. The best fidelity would be to go with creating a new empty event, just in case some user event handler saw our simulated native event and expected originalEvent to be there. In reality it's not likely to matter and the stubby little event won't have any useful properties so it won't look much like a native event. It's almost as if we're putting one there just to avoid errors.